mercantile/__init__.py

```diff
@@ -52,6 +52,19 @@ def xy(lng, lat, truncate=False):
     return x, y
 
 
+def lnglat(x, y, pi=3.141592653589793):
```

Comment:
Same precision. Unfortunately Python can't optimise imported variables like
that, so I wrote it out.
On Thu 27 Aug 2015 at 18:50 Jesse Crocker <notifications@github.com> wrote:

> In mercantile/__init__.py
> <https://github.com/mapbox/mercantile/pull/45#discussion_r38118121>:
>
> > @@ -52,6 +52,19 @@ def xy(lng, lat, truncate=False):
> >      return x, y
> >
> >
> > +def lnglat(x, y, pi=3.141592653589793):
>
> The pi arg seems out of place, because everywhere else uses math.pi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapbox/mercantile/pull/45/files#r38118121>.
>